modules/ROOT/pages/rest-api-access.adoc

```diff
@@ -54,7 +68,7 @@ For more on creating Couchbase Server users see the Couchbase Server content her
 
```

Comment:
This is a few lines down as I couldn't access the lines further up in this PR but:

```
Authenticated users will have full access to admin API functionality, application data and configuration settings, depending upon the selected role.
```

Maybe drop 'full' from 'full access' kind of sounds like its contradicting itself when you say 'have full access' but then 'depending upon role'.

Reply:
```
{enterprise} users can exert finer-grained control over functionality by using  <<lbl-rbac-roles>>
```

on the site this reads as
```users can exert finer-grained control over functionality by using RBAC Roles```

this doesn't quite read right as even non-enterprise users can use RBAC roles as you say above, they just have less to choose from. Maybe:

```
users can exert finer-grained control over functionality by using the larger selection of enterprise RBAC Roles
```
I'll let you come up with wording but imo that makes more sense